test/integrationTests/signatureHelp.integration.test.ts

```diff
@@ -87,6 +87,12 @@ suite(`SignatureHelp: ${testAssetWorkspace.description}`, function () {
         expect(c.signatures[0].parameters[c.activeParameter].label).to.equal(answer);
     });
 
+    test("Signature Help identifies active parameter based on comma for multpilte commas", async function () {
+        let c = <vscode.SignatureHelp>await vscode.commands.executeCommand("vscode.executeSignatureHelpProvider", fileUri, new vscode.Position(18, 25));
+        let answer = `double double1`;
```

Comment:
I'd inline this value on the next line. If you prefer it to be separate, I'd rename the variable to `expected`

Reply:
+1